src/Widgets/MaskedImage.vala

```diff
@@ -0,0 +1,65 @@
+/*
+* Copyright 2019 elementary, Inc. (https://elementary.io)
```

Comment:
```suggestion
* Copyright 2021 elementary, Inc. (https://elementary.io)
```

Reply:
This file comes from the notification server, so it doesn't make sense to change its copyright since there are no changes in the file